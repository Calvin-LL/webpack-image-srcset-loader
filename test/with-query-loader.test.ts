import webpack from "webpack";

import compile from "./helpers/compile";
import execute from "./helpers/execute";
import getCompiler from "./helpers/getCompiler";
import readAsset from "./helpers/readAsset";

test("use with webpack-query-loader", async () => {
  const compiler = getCompiler(
    {
      sizes: ["1x", "300w", null, "2x"],
    },
    true
  );
  const stats = await compile(compiler);

  expect(
    execute(readAsset("main.bundle.js", compiler, stats as webpack.Stats))
  ).toMatchSnapshot("result");
});
