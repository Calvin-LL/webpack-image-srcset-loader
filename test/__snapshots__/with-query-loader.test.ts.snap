// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`use with webpack-query-loader: result 1`] = `"44aacd9d8d250c420d100a3fb0f1debe.jpg 1x, f64d5a6cf45bcebd0bce0fd66010d2a9.jpg 300w, fdaf70a107ad3cebfaf1e03ff0601c1c.jpg, fdaf70a107ad3cebfaf1e03ff0601c1c.jpg 2x"`;
